parcel.go

```diff
@@ -14,47 +15,103 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("INSERT INTO parcel (client,status,address,created_at) VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		fmt.Println(err)
+		return 0, nil
```

Comment:
Если у нас ошибка, то мы должны ее вернуть, чтобы вызывающая функция узнала о ней.

Нужно пройтись по всему коду и переделать это
```suggestion
		return 0, errr
```